Gopkg.toml

```diff
@@ -72,14 +72,6 @@ required = [
 # taken from https://github.com/kubernetes/client-go/blob/kubernetes-1.11.1/Godeps/Godeps.json
```

Comment:
you can remove this comment as it referred to the deps below it. consider removing the `go-autorest` reference altogether from this file, since we don't directly depend on it.

Reply:
I thought I needed `go-autorest` in there (definitely had to do something with it to get everything to compile). Now that it is in the lockfile, doesn't appear to be required so I'll chalk it up to issues on my end.